include/solvers/navier_stokes_base.h

```diff
@@ -83,7 +84,12 @@
 #include <deal.II/numerics/solution_transfer.h>
 #include <deal.II/numerics/vector_tools.h>
 
+// Simplex
+#include <deal.II/simplex/fe_lib.h>
+#include <deal.II/simplex/quadrature_lib.h>
```

Comment:
Same comment as before if we want to be up to date with the master branch of deal

Reply:
Done!